### ISSUE TITLE
unable to run raw text with any of the pretrained model

### ISSUE THREAD

**Issue by official-dev** [danielhers/tupa#94]
Hi Team,
Great work. But when I tried to obtain the representation none of the models works. but I am getting this error. can you help me solving error which I have mentioned below

> python -m tupa example.txt -m models/mrp-20190905
> TUPA version 2.0.0 (MRP)
> Loading from 'models/mrp-20190905.json'.
> [dynet] random seed: 1
> [dynet] allocating memory: 512MB
> [dynet] memory allocation done.
> [dynet] 2.1
> Loading from 'models/mrp-20190905.enum'... Done (0.003s).
> Loading model from 'models/mrp-20190905': 169param [00:13, 12.22param/s]                                                                                                  
> tupa example.txt <stdout> --diagnostics models/mrp-20190905.diagnostics.csv --hyperparams "shared --lstm-layers 2" "amr --max-node-labels 1000 --max-edge-labels 141 --max-node-properties 1000 --max-edge-attributes 0" "dm --max-node-labels 1000 --max-edge-labels 59 --max-node-properties 510 --max-edge-attributes 0" "psd --max-node-labels 1000 --max-edge-labels 90 --max-node-properties 1000 --max-edge-attributes 0" "eds --max-node-labels 1000 --max-edge-labels 10 --max-node-properties 1000 --max-edge-attributes 0" "ucca --max-node-labels 0 --max-edge-labels 15 --max-node-properties 0 --max-edge-attributes 2" --log models/mrp-20190905.log --max-action-labels 564 --max-edge-labels 141 --max-node-properties 1000 --max-words 0 --max-words-external 0 --testscores models/mrp-20190905.test.csv
> Reading example.txt: 0 graphs [00:00, ? graphs/s]code.mrp.read(): ignoring line 0: Expecting value: line 1 column 1 (char 0)
> Reading example.txt: 1 graphs [00:00, 8905.10 graphs/s]
> Initializing: 0graph [00:00, ?graph/s]

content of the example.txt

> After graduation, Mary moved to New York City.

**Comment by danielhers** [comment]
Apologies, the README in the [mrp branch](https://github.com/danielhers/tupa/tree/mrp) is misleading. I updated it: https://github.com/danielhers/tupa/blob/mrp/README.md#parse-a-text-file
